Assets/Scripts/SortingLayers.cs

```diff
@@ -3,7 +3,12 @@
 using UnityEngine;
 
 public class SortingLayers : MonoBehaviour {
+    public bool useParent2DCollider;
     void LateUpdate() {
-        GetComponent<SpriteRenderer>().sortingOrder = -(int)(transform.position.y * 100);
+        if (useParent2DCollider) {
```

Comment:
I think better line usage here is to have the condition determine the value of a `multiplier` variable (should probably pick a better name than that) and then the last line can be
```
GetComponent<SpriteRenderer>().sortingOrder = -(int)(multiplier * 100)
```

Reply:
You're absolutely correct. I'll be making that change.